packages/drupal-themes/custom_iframe/package.json

```diff
@@ -1,5 +1,10 @@
 {
   "name": "@custom/custom_iframe",
   "version": "1.0.0",
-  "private": true
+  "private": true,
+  "scripts": {
+    "test:static": "cd ../../../apps/cms && pnpm test:static:phpcs ../../packages/drupal-themes/custom_iframe/ && pnpm test:static:phpstan ../../packages/drupal-themes/custom_iframe/",
```

Comment:
Can we create a single "test:static" command in `apps/cms` that executes both phpcs and phpstan on the same target directory?